src/NUnitTestAdapter/TestLogger.cs

```diff
@@ -19,14 +19,16 @@ namespace NUnit.VisualStudio.TestAdapter
     /// </summary>
     public class TestLogger : IMessageLogger
     {
-        private IMessageLogger _messageLogger;
+        private const string EXCEPTION_FORMAT = "Exception {0}, {1}";
 
-        private int _verbosity;
+        private IMessageLogger MessageLogger { get; set; }
+
+        private int Verbosity { get; set; }
```

Comment:
It's not something I usually do but I thought I'd try it to see how it felt. :-)